core/src/main.jl

```diff
@@ -0,0 +1,48 @@
+function help(x::AbstractString)::Cint
+    println(x)
+    println("Usage: ribasim path/to/model/ribasim.toml")
+    return 1
+end
+
+function main(ARGS::Vector{String})::Cint
```

Comment:
It would also be nice to have a `main(toml_path::AbstractString) = main([toml_path])`. Most users will use a CLI, but for testing branches ease of use for the interactive API is worth facilitating. Then we also don't need the square brackets in `examples.ipynb`.